target/linux/ath79/nand/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -18,6 +18,9 @@ case "$FIRMWARE" in
 		caldata_extract "art" 0x5000 0x844
 		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary art 0x0) 1)
 		;;
+	netgear,r6100)
```

Comment:
No valid MAC address in 0x5006?

Reply:
0x5006 contains a valid, but dummy MAC (00:03:07:12:34:56). To match the original firmware's MACs, the MAC from caldata should be taken as a base. (Actually, caldata 0x12 contains a proper MAC for the 5G interface, so we could even use that.)

Reply:
But you are using 0xc at the moment in DTS?